types/tipset.go

```diff
@@ -131,6 +131,35 @@ func (ts TipSet) MinTicket() (Signature, error) {
 	return min, nil
 }
 
+// Tickets returns the tickets in the tipset.
+func (ts TipSet) Tickets() ([]Signature, error) {
```

Comment:
Blocking: This was introduced for an incorrect reason and should be removed.

Reply:
Roger roger. Will remove and use the `Height` as @anorth describes.

From an above comment:

>  the implementation actually catches up with the spec and includes one ticket per preceding null block. Right now this doesn't happen and all blocks have a single ticket.

Do we have an issue to track this? Seems like a correctness issue.

Reply:
Yes.  This change is tracked [here](https://github.com/filecoin-project/go-filecoin/issues/2222) as part of [this epic](https://github.com/filecoin-project/go-filecoin/issues/2412).